src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/BasicResultProviderTest.vbproj

```diff
@@ -19,6 +19,8 @@
     <RemoveIntegerChecks>true</RemoveIntegerChecks>
     <RestorePackages>true</RestorePackages>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <VBRuntime>
+    </VBRuntime>
```

Comment:
Yeah, this is actually overriding the shared targets so that we'll include the full VB runtime (rather than VB Core).  I need it in the tests for manipulating IDMOs.  I was going to add a comment, I decided not to.  If someone removes this, they'll get a compile error if there is any code that needs it (and who knows, at some point, we might refactor these tests to not need the latebinder, at which point, I wouldn't want the comment to scare people from removing this).